app/controllers/announcements_controller.rb

```diff
@@ -0,0 +1,41 @@
+class AnnouncementsController < ApplicationController
+  before_action :authenticate_user!
+  
+  def new
+    @announcement = Announcement.new
+    @licensees = Licensee.all
+    @placements = Placement.all
+  end
+  
+  def create
+    #byebug
+    Announcement.create(
+      user_id: params[:announcement][:user_id],
+      placement_id: params[:announcement][:placement_id],
```

Comment:
It would be better to use the strong parameters approch here, via Announcement.create(params[:announcement])